Automattic/wp-calypso
# AB Test selling G Suite Business not Basic in new domain flow

DO NOT MERGE TILL ADDING ADDITIONAL USERS AND LISTING USERS IS TESTED

#### Changes proposed in this Pull Request

* AB Tests selling G Suite Business not Basic in new domain flow

#### Testing instructions

* Add yourself to Business AB Test:  `localStorage.setItem( 'ABTests', '{"gSuitePlan_20190117":"business"}' );`
* Goto purchase new domain
* Observe that storage copy changes
* Observe pricing changes
* Continue to checkout
* Does checkout work?
* You should see G Suite Unlimited in your purchase history

![screen shot 2019-01-17 at 4 27 48 pm](https://user-images.githubusercontent.com/6817400/51349971-f75b6000-1a74-11e9-8e49-c6aca73fe000.png)
